### DISCUSSION TITLE
Is there a to add my theme to the official theme list ?

### DISCUSSION THREAD

**Discussion by mehedi-codes** [JanDeDobbeleer/oh-my-posh#4177] [Themes]
[![dualsimplicity](https://github.com/JanDeDobbeleer/oh-my-posh/assets/89072438/1906681b-3e41-45e8-9b99-d3aebee46ed6)](https://github.com/devbytemehedi/dualsimplicity-ohmyposh-theme)

Theme Name : DualSimplicity 
Source code : https://github.com/devbytemehedi/dualsimplicity-ohmyposh-theme

**Comment by JanDeDobbeleer** [comment]
@devbytemehedi hey, thanks for sharing this one! No, we're no longer adding them to the repo. You can also share on our Discord though.

**Reply by mehedi-codes** [reply]
@JanDeDobbeleer thanks for your response. i will share this on Discord.
